gammapy/background/models.py

```diff
@@ -4,13 +4,23 @@
 from __future__ import print_function, division
 import numpy as np
 from astropy.modeling.models import Gaussian1D
+from astropy.units import Quantity
+from astropy.coordinates import Angle
+from astropy.io import fits
+from astropy.table import Table
+from astropy import wcs
 
-__all__ = ['GaussianBand2D']
+__all__ = ['GaussianBand2D',
+           'CubeBackgroundModel',
+           'make_linear_bin_edges_arrays_from_wcs',
```

Comment:
I'd say don't make these helper functions public for now, i.e. remove them from this `__all__` list.
Should we need them in a second place in the future, we'll move them to a better place (e.g. `image.utils` or `utils.something`) and make them public.